.github/workflows/scala.yml

```diff
@@ -56,6 +56,12 @@ jobs:
         uses: DeLaGuardo/setup-graalvm@2.0
         with:
           graalvm-version: ${{ env.graalVersion }}.${{ env.javaVersion }}
+      - name: Set Up Native Image (Windows)
```

Comment:
[This action](https://github.com/marketplace/actions/setup-graalvm-ce) can set up GraalVM and install native image.